typings/helpers.d.ts

```diff
@@ -45,7 +49,7 @@ declare global {
             leafletMissionPositionMarkerAdd(poi: POI): void;
         };
         map_filters_service: {
-            initialize(a: MapFilterInitializer): void;
+            // initialize(a: MapFilterInitializer): void;
```

Comment:
Remove if not needed

Reply:
commented because will be re-needed later for centerMap dynamic mode